examples/camera-app/linux/src/clusters/camera-avstream-mgmt/camera-av-stream-manager.cpp

```diff
@@ -155,6 +151,23 @@ Protocols::InteractionModel::Status CameraAVStreamManager::VideoStreamAllocate(c
     return Status::DynamicConstraintError;
 }
 
+void CameraAVStreamManager::OnVideoStreamAllocated(const VideoStreamStruct & allocatedStream, bool shouldStartNewVideo)
+{
+    // Only start the video stream if the flag indicates we should (new allocation or configuration changed)
+    if (shouldStartNewVideo)
```

Comment:
If the stream is getting re-used, then there is an already active stream. If that was modified, then the stream needs to be stopped and restarted with the new parameters.

Reply:
Acked